templates/Wpf/Features/XamlIslandUwpApp/Param_ProjectName.XamlIsland/Properties/AssemblyInfo.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+// General Information about an assembly is controlled through the following
+// set of attributes. Change these attribute values to modify the information
+// associated with an assembly.
+[assembly: AssemblyTitle("Param_ProjectName.XamlIsland")]
+[assembly: AssemblyDescription("")]
+[assembly: AssemblyConfiguration("")]
+[assembly: AssemblyCompany("")]
+[assembly: AssemblyProduct("Param_ProjectName.XamlIsland")]
+[assembly: AssemblyCopyright("Copyright ©  2020")]
```

Comment:
year should be set with a replacement during generation. Also elsewhere

Reply:
done